### PR TITLE
Don't output some unsupported tags or attributes

### PR DESCRIPTION
**Pull request by sebbacon**

### CODE REVIEW

**Review by khaydarov — APPROVED** [review]

File: htmlslacker/htmlslacker.py [hunk 1]
```diff
@@ -66,14 +70,17 @@ def handle_endtag(self, tag):
             self.output += '>'
         if tag == 'code':
             self.output += '`'
+        if tag == 'style' or tag == 'script':
```
Comment (khaydarov) [inline htmlslacker/htmlslacker.py]: I think it would be better to use `else`

Comment (khaydarov) [reply htmlslacker/htmlslacker.py]: @sebbacon

Comment (sebbacon) [reply htmlslacker/htmlslacker.py]: Not sure what you mean - if you mean any tag that is not explicitly handled should result in the text content being dropped, that would mean `h1` text would get dropped, for example, or anything in a `<span>`, or things in any semantic tags like `<header>` or whatever? 

Or if you mean these should all be `elif` switches, then sure :)

**Review by n0str — APPROVED** [review]

File: htmlslacker/htmlslacker.py (hunk 1 above)
Comment (khaydarov) [reply htmlslacker/htmlslacker.py]: Yes, I meant any tag which is not supported by slack should be dropped, but maybe it is not correct, better to output only content.

**Comment by khaydarov** [comment]
thanks @sebbacon

**Pull request merged**